app/Jobs/ProcessRecurrings.php

```diff
@@ -9,6 +9,7 @@
 use Illuminate\Foundation\Bus\Dispatchable;
 use App\Recurring;
 use App\Spending;
+use Illuminate\Support\Facades\Log;
```

Comment:
..?

Reply:
Jip. That was just used for logging during debugging. Just fixed that. ;)